act-rules-format.bs

```diff
@@ -188,7 +182,7 @@ ACT Rules can be used to test accessibility requirements that are not part of a
 
 If the `failed` outcome can not be mapped to *not satisfied* for an accessibility requirement, that requirement MUST NOT be included in the accessibility requirements mapping. The optional [Background section](#background) could be used to list accessibility requirements and standards that may be thematically related, but for which the rule is not a failure test.
```

Comment:
Misuse of "may" keyword. Here some suggestions:

- could be used to list accessibility requirements and standards that are thematically related (remove "may")
- could be used to list accessibility requirements and standards when they are thematically related (replace "may")
- MAY be used to list accessibility requirements and standards that are thematically related (move "may" to make it a requirement)